### ISSUE TITLE
Dashlets Reports: Same Multiple Field (DateTime type) conditions in Parameters are rendered broken 

### ISSUE THREAD

**Issue by gunnicom** [SuiteCRM/SuiteCRM#5696]
Still present in 7.11.7

Steps to reproduce...

1. Create a Report with the Report Module = Meetings which has multiple conditions of the same DateTime field. E.g. Start Date
![Screenshot_2019-08-02 SuiteCRM(4)](https://user-images.githubusercontent.com/4041275/62379844-a9550800-b53f-11e9-9210-88cc76ce1ca7.png)
2. Mark these conditions are Parameters
3. Save Report and navigate to Dashboard
4. Add a Report Dashlet
5. Edit Report Dashlet to reference the newly created Report and Save
6. Re-Edit the Report Dashlet and see the secondary condition is rendered in complete

Actual Behaviour
![Screenshot_2019-08-02 SuiteCRM(3)](https://user-images.githubusercontent.com/4041275/62379736-7a3e9680-b53f-11e9-856d-d4056be57ea7.png)

Expected behaviour.

The DateTime conditions/parameters should render the same as they do on the Detail View of the Report and editable to report accurately.
![Screenshot_2019-08-02 sss » Reports » SuiteCRM](https://user-images.githubusercontent.com/4041275/62379802-92161a80-b53f-11e9-8885-a88c351ae299.png)

After Update from 7.8.10 to 7.8.17 (Still in 7.8.18)
A picture is more than thousand words. Same field, twice as parameter:

![grafik](https://user-images.githubusercontent.com/550251/38564743-08108baa-3ce0-11e8-9941-9aea40426cc1.png)

![grafik](https://user-images.githubusercontent.com/550251/38564815-2d9631d6-3ce0-11e8-9d4f-a0f8056dd567.png)

**Comment by gunnicom** [comment]
Reproducable in 7.10.4.
Steps to reproduce:
- Create a Report
- Report Module: Meetings
- Add "Start Date" twice

**Comment by pgorod** [comment]
I believe this is a duplicate of #5663, and I've seen several people complaining.

**Comment by gunnicom** [comment]
Yes, you may be right. Could be a duplicate.

**Comment by gunnicom** [comment]
Bug is still there for dashlets after update to 7.8.19 :(

**Comment by gunnicom** [comment]
Still present in 7.8.22

**Comment by gunnicom** [comment]
Bug still present in 7.8.27

**Comment by Dillon-Brown** [comment]
This issue has been closed as version 7.8.x of SuiteCRM has reached its end of life and therefore will no longer be receiving updates. We have also been unable to replicate this issue in any newer version of SuiteCRM such as 7.10.x or 7.11.x however if you are able, please leave a comment explaining how and we'll re-open this issue.

For more information about 7.8.x end of life please visit our [blog post](https://suitecrm.com/7-8-x-end-of-life/)

**Comment by gunnicom** [comment]
That of course is the easiest way to get rid of uncomfortable bug reports. Just wait long enough till a version reaches EOL :(

**Comment by pgorod** [comment]
I guess this is the important part:

> We have also been unable to replicate this issue in any newer version

If the bug is still present in the latest version, of course the issue should remain open.

**Comment by samus-aran** [comment]
> That of course is the easiest way to get rid of uncomfortable bug reports. Just wait long enough till a version reaches EOL :(

Actually @gunnicom, closing these issues have an element of comfortability for us aswell. We certainly don't like having these feelings of not resolving issues for community members. The decision to close these issues wasn't taken lightly but a necessity. It will be unmaintainable for the core team to keep these open in the hope to someday resolve when we have other versions to support.

**Comment by samus-aran** [comment]
I personally went to verify this issue and confirm it still present in the 7.11.x branch.
That shouldn't been a struggle to reproduce it but it was. 

Everyone should strive to have fuller report bugs. Incomplete ones only make it harder for anyone (core or community members) who want to help less likely to understand or in this case reproduce it accurately. 

I will update the title and the bug report to reflect.

**Comment by gunnicom** [comment]
Yeah it is me to blame for doing bad reports. Sorry that i do not want to put too much effort in this, if i do not know if it gets addressed or just closed some years later.

**Comment by samus-aran** [comment]
> Yeah it is me to blame for doing bad reports. Sorry that i do not want to put too much effort in this, if i do not know if it gets addressed or just closed without being fixed some years later.

Ok.... so, its a chicken and egg scenario? We can't fix faster if we have to spend more time figuring out the issue in the first place. But you don't want to utilise the steps we provide (the issue template) because you are making a point not to? By not providing a fuller bug report you are only making it more difficult for ANYONE to help (not just the core team).

It obviously showed my personal frustration that we closed an issue prematurely, so I apologise for that. But my point still stands, regardless if an issue is 1 weeks old or years old, everyone should strive to have fuller reports so anyone can try and contribute.
